yaml_edit/example/example.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:yaml_edit/yaml_edit.dart';
```

Comment:
License header, example:
https://github.com/google/dart-neats/blob/73ada3d69f092cef445dd54e4e79953f8af3b969/safe_url_check/example/main.dart#L1-L13

This goes for all files.